doc/articles/uno-development/Uno-UI-Layouting-iOS.md

```diff
@@ -6,7 +6,7 @@ uid: Uno.Contributing.LayoutingiOS
 
 The layouting cycle (measure and arrange) in Uno on iOS is a mingling of native layouting logic and logic in managed code. These interactions are summarized in the diagram below. This information is primarily intended to help when debugging Uno, but may also be useful when attempting to incorporate non-Uno views into the visual tree.
 
-``` mermaid
```

Comment:
When the screensize is smaller, the diagrams are still too much offsetted 🤔 
![image](https://github.com/unoplatform/uno/assets/78549750/cc1950c3-ab6c-44ce-ae26-572ef4876b61)

Reply:
@morning4coffe-dev thanks for the feedback! I am not able to replicate on my side.
Can you give me more details (browser+version used) and make sure to have the proper azure static web app link related to the PR when testing just to make sure this is not the issue here please.
![image](https://github.com/unoplatform/uno/assets/16295702/7cfb92cb-fa68-4e72-8281-4123632bd4fb)
![image](https://github.com/unoplatform/uno/assets/16295702/f8ec83a9-9ee3-4493-9e0a-7d2d835a890b)

Reply:
Hmmm... interesting. Using `Edge stable - 126.0.2592.24` on my side. Let me test it a bit more

Reply:
It happened to me again when I opened the page, but when I tried to reload it's working well 🤔
https://github.com/unoplatform/uno/assets/78549750/b38bb9de-2b8e-4233-9706-f525bcd12442

Reply:
I tried using Edge, but I'm still not able to reproduce it on my end. @morning4coffe-dev since it was working after a refresh on your side maybe it was just a cache issue? Can you try to clean your cache and re-try again please?

Reply:
Good idea, I tried hard resetting and indeed I am not able to repro the issue again 👍 Resolving the comment now.

Reply:
Thanks a lot for the confirmation @morning4coffe-dev, appreciated!